multi_import/formats.py

```diff
@@ -167,16 +168,16 @@ def export_set(self, dataset):
         # By default use the C-based CSafeDumper,
         # otherwise fallback to pure Python SafeDumper.
         if CSafeDumper:
-            return _yaml.yaml.dump(
-                dataset._package(ordered=False),
```

Comment:
Question, is dropping the ordered param here going to result in any unexpected behaviour? Probably not a big deal, but it might start providing exports in a different sorted order than previously expected.

Reply:
I believe this simply let's the the dumper know the data set isn't ordered. As far as the order, i assume it would fallback to the ordering parameter set by the queryset or the model meta attribute itself.

Its worth investigating though, perhaps run a few exports and see if the order changes?

Reply:
Sounds good, will run a couple extra checks on this

Reply:
Well, trying to reintroduce that arg throws an error for unexpected keyword, so I guess we shouldn't worry about it!